functions/bigtable/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "nodejs-docs-samples-functions-bigtable",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "test": "mocha test/*.test.js --timeout=20000",
+    "start": "functions-framework --target=get"
+  },
+  "dependencies": {
+    "@google-cloud/bigtable": "^2.3.2",
+    "@google-cloud/functions-framework": "^1.5.1"
+  },
+  "devDependencies": {
+    "mocha": "^7.0.0",
+    "proxyquire": "^2.1.0",
+    "sinon": "^9.0.0"
+  },
+  "cloud-repo-tools": {
```

Comment:
Looks awesome! 

Tiny nit: we don't need the `cloud-repo-tools` section in package.json, because we don't use `repo-tools` for system tests here.